core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-scheduling/dot-experiments-configuration-scheduling.component.scss

```diff
@@ -10,3 +10,13 @@ h2 {
 .scheduling-setup-date-spacing {
     margin: 0 $spacing-1;
 }
+
+:host ::ng-deep {
+    dot-icon {
+        color: $gray-light;
+
+        &.isDone {
+            color: $success;
```

Comment:
This is a legacy color, check `libs\dotcms-scss\shared\_colors.scss` for the new variable